src/test/java/com/o19s/es/ltr/feature/store/index/IndexFeatureStoreTests.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright [2017] Wikimedia Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.o19s.es.ltr.feature.store.index;
+
+import com.o19s.es.ltr.LtrTestUtils;
+import com.o19s.es.ltr.feature.store.StorableElement;
+import com.o19s.es.ltr.feature.store.StoredFeature;
+import com.o19s.es.ltr.feature.store.StoredFeatureSet;
+import com.o19s.es.ltr.feature.store.StoredLtrModel;
+import org.apache.lucene.util.LuceneTestCase;
+import org.apache.lucene.util.TestUtil;
+import org.elasticsearch.client.Requests;
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.xcontent.NamedXContentRegistry;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentFactory;
+import org.elasticsearch.common.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.o19s.es.ltr.feature.store.index.IndexFeatureStore.STORE_PREFIX;
+import static com.o19s.es.ltr.feature.store.index.IndexFeatureStore.indexName;
+import static com.o19s.es.ltr.feature.store.index.IndexFeatureStore.isIndexStore;
+import static com.o19s.es.ltr.feature.store.index.IndexFeatureStore.storeName;
+import static org.apache.lucene.util.TestUtil.randomRealisticUnicodeString;
+import static org.apache.lucene.util.TestUtil.randomSimpleString;
+import static org.hamcrest.CoreMatchers.equalTo;
+
+public class IndexFeatureStoreTests extends LuceneTestCase {
+
+    public void testParse() throws Exception {
+        parseAssertions(LtrTestUtils.randomFeature());
+        parseAssertions(LtrTestUtils.randomFeatureSet());
+        parseAssertions(new StoredLtrModel(
+                randomSimpleString(random(), 5, 10),
+                LtrTestUtils.randomFeatureSet(),
+                randomSimpleString(random(), 5, 10),
+                randomRealisticUnicodeString(random(), 5, 1000),
+                true));
+    }
+
+    public void testIsIndexName() {
+        assertTrue(isIndexStore(IndexFeatureStore.DEFAULT_STORE));
+        assertFalse(isIndexStore("not_really"));
+        assertFalse(isIndexStore(IndexFeatureStore.STORE_PREFIX));
+        for (int i = 0; i < random().nextInt(10) + 10; i++) {
```

Comment:
`random().nextInt(10) + 10` is evaluated for each step of the loop. It is technically correct, but it feels a bit strange to have a non constant bound to a loop (feel free to ignore).

Same comment applies to `testStoreName()`.